packages/block-library/src/block/edit.js

```diff
@@ -117,6 +118,16 @@ function applyInitialOverrides( blocks, overrides = {}, defaultValues ) {
 	} );
 }
 
+function isAttributeEqual( attribute1, attribute2 ) {
+	if (
+		attribute1 instanceof RichTextData &&
+		attribute2 instanceof RichTextData
+	) {
+		return attribute1.toString() === attribute2.toString();
```

Comment:
A bit of a gotcha here that if the attributes are different `RichTextData` instances then `===` won't be true even though we expect them to be most of the time. 🤔

Reply:
I can't really think of another way, so this looks ok to me.